src/main/java/gillycarpetaddons/mixins/PhantomSpawnerMixin.java

```diff
@@ -68,7 +76,7 @@ private boolean isSpectatorOrAtMobCap(PlayerEntity instance) {
     private void isInMooshromBiome(ServerWorld instance, Entity entity) {
         BlockPos pos=new BlockPos(entity.getPos());
         RegistryEntry<Biome> registryEntry = instance.getBiome(pos);
-        if(GillyCarpetAddonsSettings.disablePhantomSpawningInMushroomFields && registryEntry.getKey().get() != BiomeKeys.MUSHROOM_FIELDS){
+        if(!GillyCarpetAddonsSettings.disablePhantomSpawningInMushroomFields || registryEntry.getKey().get() != BiomeKeys.MUSHROOM_FIELDS){
```

Comment:
Looks like a bug we missed 😲

I'll create a bug for this so we can note it in the pre-release version it affects.